xbmc/video/windows/GUIWindowFullScreen.cpp

```diff
@@ -473,3 +474,11 @@ bool CGUIWindowFullScreen::HasVisibleControls()
 {
   return m_controlStats->nCountVisible > 0;
 }
+
+int CGUIWindowFullScreen::GetOSDWindowID()
```

Comment:
this looks like an ugly hack and if requirements for VP and RP diverge, we'll need even more of them. better create a new class for RP.